Assets/Scripts/Framework/Buttons/PointsSystem/AddPointsButton.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+namespace ProefExamen.Framework.Buttons.PointsSystem
+{
+    /// <summary>
+    /// A class that adds points to the points system when the button is pressed for testing purposes.
+    /// </summary>
+    public class AddPointsButton : BasicButton
+    {
+        [SerializeField]
+        private int _pointsToAdd = 1;
+
+        /// <summary>
+        /// <inheritdoc/>
+        /// </summary>
+        public override void OnButtonPressed() => Framework.PointsSystem.PointsSystem.Points += _pointsToAdd;
```

Comment:
I can't check it now, but this line looks pretty long. Is it longer than 120 characters?

Reply:
This is within the 120 characters limit.

Reply:
👍